gcp-function-http/src/main/java/io/micronaut/gcp/function/http/GoogleFunctionHttpRequest.java

```diff
@@ -130,22 +138,43 @@ final class GoogleFunctionHttpRequest<B> implements
             B built = parsedBody != null ? parsedBody :  (B) bodyBuilder.buildBody(this::getInputStream, this);
             return Optional.ofNullable(built);
         });
+        this.byteBody = SupplierUtil.memoized(() -> {
+            try {
+                return InputStreamByteBody.create(
+                    googleRequest.getInputStream(),
```

Comment:
when/how is this closed?

Reply:
it's not, but i assume it doesn't really matter for functions